components/scripts/lib/gradle.sh

```diff
@@ -13,7 +13,7 @@ invoke_gradle() {
   local init_scripts_dir
   init_scripts_dir="$(init_scripts_path)"
 
-  pushd "${project_dir}" > /dev/null 2>&1 || die "ERROR: The subdirectory ${project_dir} (set with --project-dir) does not exist in ${project_name}." 3
+  pushd "${project_dir}" > /dev/null 2>&1 || die "ERROR: The subdirectory ${project_dir} (set with --project-dir) does not exist in ${project_name}."
```

Comment:
For maven, this is defined as a INVALID_INPUT

Reply:
✅ Done